plugins/auth-backend/config.d.ts

```diff
@@ -177,11 +140,6 @@ export interface Config {
           callbackUrl?: string;
         };
       };
-      /** @visibility frontend */
-      awsalb?: {
-        iss?: string;
```

Comment:
Looking in aws-alb provider, [`issuer` is used instead of `iss`](https://github.com/backstage/backstage/blob/ed4cd84b10ae123882b0cf549ea9f541ae7d843d/plugins/auth-backend-module-aws-alb-provider/src/authenticator.ts#L38). This change has been reflected in c52052b